sql/core/src/test/scala/org/apache/spark/sql/execution/metric/SQLMetricsSuite.scala

```diff
@@ -396,7 +396,9 @@ class SQLMetricsSuite extends SharedSparkSession with SQLMetricsTestUtils
       }
   }
 
-  test("SPARK-32629: ShuffledHashJoin(full outer) metrics") {
+  // We adjust building's memory settings and it seems affecting this test.
```

Comment:
For completeness, I would file a JIRA for this one though. e.g.) TODO(SPARK-XXXXX): Reenable this blah blah